src/components/organisms/ListAdd/ListAdd.tsx

```diff
@@ -0,0 +1,39 @@
+import { useAppDispatch } from "store/hooks";
+import { ListColors, ListContent } from "types/type";
+
+import ListCreator from "../ListCreator/ListCreator";
+import { addList, setShowModal } from "../../../features/list/listSlice";
+
+const ListAdd = () => {
+  const dispatch = useAppDispatch();
+
+  const handleCloseModal = () => {
+    dispatch(setShowModal());
+  };
+
+  const addNewList = async (newName: string, newColor: ListColors) => {
+    try {
+      if (newName !== "") {
+        const newList: ListContent = {
+          name: newName,
+          color: newColor,
+        };
+
+        const data = await dispatch(addList(newList));
+        if (data) {
+          handleCloseModal();
+        }
+      }
+    } catch (err) {}
+  };
```

Comment:
Gdzie jest obsługa errora?

Reply:
Teoretycznie jakby był błąd to byłby chyba w storze złapany to nie wiem czy mam ten try catch usunąć?

Reply:
jeśli jest łapany w store to ten try/catch jest tutaj niepotrzebny